src/components/icons/Icon.jsx

```diff
@@ -601,7 +601,10 @@ const Icon = ({
           aria-labelledby={labelledBy}
           focusable="false"
         >
-          <title id={titleId}>{title || titleFallback}</title>
+          <text id={titleId} hidden>
+            {title || defaultTitle}
```

Comment:
i think this condition will break intention when I set `title=""`. `??` instead?

Reply:
Could you explain why we want the title to be empty? `title=""` makes Icon inaccessible, it doesn't remove it from the accessibility tree. If you want to hide it, `arria-hidden` has to be added. I don't see any other intention to do that.

Reply:
From discussion on deque:
<img width="727" alt="Screenshot 2022-03-10 at 18 51 35" src="https://user-images.githubusercontent.com/60467496/157724677-00cf01d1-510c-4cbe-b4ef-e3f9bf1c2989.png">

Reply:
It's not about accessibility. As you mentioned for that case I should use aria-hidden. 

We need to provide a way to hide native tooltip. There are 2 cases
- we don't want any tooltip on a element
- we provide custom tooltip solution and label

in both cases I may apply a11y label on an outer element to satisfy accessibility requirement and not be forced to display native tooltip at the same time.